api/src/zero/module/wgs.clj

```diff
@@ -156,66 +160,66 @@
 
 (defn really-submit-one-workflow
   "Submit IN-GS for reprocessing into OUT-GS in ENVIRONMENT."
-  [environment in-gs out-gs]
+  [environment in-gs out-gs sample]
   (let [path (wdl/hack-unpack-resources-hack (:top workflow-wdl))]
     (cromwell/submit-workflow
      environment
      (io/file (:dir path) (path ".wdl"))
      (io/file (:dir path) (path ".zip"))
-     (make-inputs environment out-gs in-gs)
+     (make-inputs environment out-gs in-gs sample)
      (util/make-options environment)
      cromwell-label-map)))
 
-(defn submit-workflow
-  "Submit OBJECT from IN-BUCKET for reprocessing into OUT-GS in
-  ENVIRONMENT."
-  [environment in-bucket out-gs object]
-  (let [in-gs (gcs/gs-url in-bucket object)]
-    (really-submit-one-workflow environment in-gs out-gs)))
-
-(defn submit-some-workflows
-  "Submit up to MAX workflows from IN-GS to OUT-GS in ENVIRONMENT."
-  [environment max in-gs out-gs]
-  (let [bucket (all/readable! in-gs)]
-    (letfn [(input? [{:keys [name]}]
-              (let [[_ unsuffixed suffix] (all/bam-or-cram? name)]
-                (when unsuffixed [unsuffixed suffix])))]
-      (let [slashified (all/slashify out-gs)
-            done   (set/union (all/processed-crams slashified)
-                              (active-objects environment in-gs))
-            suffix (->> in-gs
-                        gcs/parse-gs-url
-                        (apply gcs/list-objects)
-                        (keep input?)
-                        (into {}))
-            more   (->> suffix
-                        keys
-                        (remove done)
-                        (take max)
-                        (map (fn [base] (str base (suffix base)))))
-            submit (partial submit-workflow environment bucket slashified)
-            ids    (map submit more)]
-        (run! prn ids)
-        (vec ids)))))
-
-(defn submit-some-workflows-or-throw
-  "Submit up to MAX-STRING workflows from IN-GS to OUT-GS in ENVIRONMENT."
-  [environment max-string in-gs out-gs]
-  (let [max (util/is-non-negative! max-string)]
-    (submit-some-workflows environment max in-gs out-gs)))
-
-(defn run
-  "Reprocess the BAM or CRAM files described by ARGS."
-  [& args]
-  (try
-    (let [env (zero/throw-or-environment-keyword! (first args))]
-      (apply (case (count args)
-               4 submit-some-workflows-or-throw
-               3 (partial all/report-status cromwell-label)
-               (throw (IllegalArgumentException.
-                       "Must specify 3 or 4 arguments.")))
-             env (rest args)))
-    (catch Exception x
-      (log/error x)
-      (log/debug description)
-      (throw x))))
+;(defn submit-workflow
```

Comment:
please delete unused/commented out code. We can resurrect it from the history if needed.

Reply:
I made the change you suggested @ehigham!